Project/src/MakeCall/CallCard.js

```diff
@@ -781,6 +864,21 @@ export default class CallCard extends React.Component {
                                         <Icon iconName="People" />
                                     }
                                 </span>
+                                <span className="in-call-button "
```

Comment:
Can we render these only if it is CTE? Or the function inside to return diff items for CTE/ACS if the features are supported on one of the platforms

Reply:
The features are supported in both ACS and CTE